hw/formal/README.md

```diff
@@ -252,23 +252,27 @@ can be used to add coverpoints to your design, where the cover
 property uses the same SVA syntax, operators, and system functions as the the
 assert properties.
 
-## Running FPV
+## Running FPV on DVSim
 
 ### Cadence JapserGold
 If you have access to JasperGold from Cadence, you can formally verify your
-assertions. For example, to run formal property verification (FPV) using
+assertions.
+Before running FPV, please make sure the target has been added to the [batch script](
+https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson)
+
+For example, to run formal property verification (FPV) using
 JasperGold on module `gpio`, type:
```

Comment:
can you format this as one sentence per line?

Reply:
Thanks for catching that!